modules/guides/pages/create-user-defined-function.adoc

```diff
@@ -1,24 +1,24 @@
 = Creating a User-Defined Function
-:description: Creating a N1QL User-Defined Function to call your JavaScript function.
+:description: Creating an {sqlpp} User-Defined Function to call your JavaScript function.
 :page-edition: Enterprise Edition
 :page-toclevels: 2
-:keywords: N1QL, UDF, user-defined function
+:keywords: N1QL, UDF, user-defined function, SQL++
 [abstract]
 {description}
 
 == Introduction
 
 [#creating-the-n1ql-udf-function]
-== Creating the N1QL User-Defined Function
+== Creating the {sqlpp} User-Defined Function
 
-Before you can run the JavaScript function you have created (see xref:create-javascript-library.adoc[]), you will need to create a N1QL User-Defined Function to reference it. 
-You can create a N1QL User-Defined Function by using:
+Before you can run the JavaScript function you have created (see xref:create-javascript-library.adoc[]), you will need to create an {sqlpp} User-Defined Function to reference it.
+You can create an {sqlpp} User-Defined Function by using:
```

Comment:
See comments on previous pull request. If we specify that "SQL++" is pronounced "sequel plus plus", we don't have to change the indefinite article.

Google ngram viewer gives [more matches for "a SQL" than "an SQL"](https://books.google.com/ngrams/graph?content=a+SQL%2Can+SQL&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Ca%20SQL%3B%2Cc0%3B.t1%3B%2Can%20SQL%3B%2Cc0), especially in recent years.